charts/azure-apps/values.yaml

```diff
@@ -0,0 +1,19 @@
+# secretsStoreCsiDriverAzure -- [secrets-store-csi-driver-provider-azure](https://github.com/Azure/secrets-store-csi-driver-provider-azure) ([example](./examples/secrets-store-csi-driver-provider-azure.yaml))
+# @default -- -
+secretsStoreCsiDriverAzure:
+  # secretsStoreCsiDriverAzure.enabled -- Enable secrets-store-csi-driver-provider-azure
+  enabled: false
+  name: secrets-store-csi-driver-provider-azure
```

Comment:
perhaps we should rather go for a naming concept like vault did:

https://github.com/hashicorp/vault-csi-provider

so: `azure-key-vault-csi-provider`  or `azure-kv-csi-provider` 

What are your thoughts? Also on the Namespace and Project name.

Reply:
Yes, I think the same, also, the shortest the better, so we can proceed with `azure-kv-csi-provider`. I will update, thanks for the suggestion.